File: src/main/kotlin/io/github/hanchen/hcmattersplit/inventory/InventoryUtil.java
package com.hanchen.hcfenjie.inventory;

import com.hanchen.hcfenjie.Main;
import org.bukkit.Bukkit;
import org.bukkit.entity.Player;
import org.bukkit.inventory.Inventory;
import org.bukkit.inventory.InventoryHolder;


public class InventoryUtil {
    public static void openInventory(Player player) {
        Inventory inventory = Bukkit.createInventory((InventoryHolder) null, 54, Main.getInstance().inventoryTitle);
        inventory.setItem(49, Main.getInstance().inventoryItemStack);
        player.openInventory(inventory);
    }
}

File: src/main/kotlin/io/github/hanchen/hcmattersplit/inventory/InventoryUtil.kt
package io.github.hanchen.hcmattersplit.inventory

import io.github.hanchen.hcmattersplit.Main
import org.bukkit.Bukkit
import org.bukkit.entity.Player
import org.bukkit.inventory.Inventory

fun Player.openInventory() {
    val inventory = Bukkit.createInventory(null, 54, Main.plugin.inventoryTitle).apply {
        setItem(49, Main.plugin.inventoryItemStack)
    }
    openInventory(inventory)
}